src/services/news/index.jsx

```
@@ -0,0 +1,314 @@
+                onClick={isLoading ? null : handleSubmit}
+                spinner={isLoading ? <Spinner size="sm" /> : null}
+              />
+            </div>
+          </div>
+        )}
+      </Form>
+    </>
+  );
+};
+
+export const NewsPreviewer = ({ image, isOpen, onClose, isTop = false }) => {
```

ESLint

[error] 'image' is missing in props validation

(react/prop-types)